pkg/job_controller/api/v1/types.go

```diff
@@ -57,6 +61,24 @@ type JobStatus struct {
 // own set of ReplicaTypes.
 type ReplicaType string
 
+// ElasticScalingStatus represents the current elastic scaling status of the training job.
+type ElasticScalingStatus struct {
+	// The number of current scaling pod replicas.
+	CurrentReplicas int32 `json:"currentReplicas,omitempty"`
+
+	// The number of last scaling pod replicas.
+	LastReplicas int32 `json:"lastReplicas,omitempty"`
+
+	// Continue represents whether the job needs to continue scaling.
+	Continue bool `json:"continue,omitempty"`
```

Comment:
will user amend this `continue` field manually?  if not, who will pause the scaling progress

Reply:
The torchelastic controller will amend this continue field. When continue is set to false in the last control loop, the controller will pause the scaling progress.